Fixtures/DIContainer.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
Should we strip out the specific of AdvancedCache from this before adding it to the common?

Reply:
The idea of this particular fixture is to keep adding concretes to it. As you need a concrete, add it here. That was the original design.

But we could separate that out and move it into each individual test where it's needed.

What do you think?

Reply:
Same line of thinking here than in my other comments :)

Reply:
Okay. I'll remove the concrete from here. We'll add it into the specific tests.

Reply:
Done here https://github.com/wp-media/phpunit-wp-rocket/pull/1/commits/45bed44cef0ebcdf50f262c9a8cbdad136567ae4